### ISSUE TITLE
Upgrade unocss version to "0.50.5", unable to start project, terminal crash

### ISSUE THREAD

**Issue by Qingmao-w** [unocss/unocss#2857]
### UnoCSS version

0.50.5

### Describe the bug

"unocss": "0.50.5",

If the version of unocss is below <= "0.50.4", the npm run dev will start normally, and if you upgrade to 0.50.5, you will not be able to start the project. The terminal will crash .

Github warehouse link below.
Reproduce steps:
1. Modify unocss version to 0.50.4, execute "pnpm install", execute "pnpm run dev", and the project will start normally,
2. Modify unocss version to 0.50.5, execute "pnpm install", execute "pnpm run dev", project cannot start, terminal crashes

### Reproduction

https://github.com/Wxw-Gu/demoUnocss

### Validations

- [X] Read the [Contributing Guidelines](https://github.com/unocss/unocss/blob/main/CONTRIBUTING.md).
- [X] Check that there isn't [already an issue](https://github.com/unocss/unocss/issues) that reports the same bug to avoid creating a duplicate.
- [X] Check that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/unocss/unocss/discussions) or join our [Discord Chat Server](https://chat.antfu.me).
- [X] The provided reproduction is a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by Qingmao-w** [comment]
> It seems to be fine on Stackblitz https://stackblitz.com/~/github.com/Wxw-Gu/demoUnocss, can you provide more info?

The current unocss in pageage.json is "0.50.4" and needs to be modified to "0.50.5". It has crashed
![error](https://github.com/unocss/unocss/assets/120553894/4e5dc30a-fccb-4a1e-be97-b0acabb3003e)

**Comment by antfu** [comment]
Wait, we are on 0.53.5 now, why you are still on 0.50?

**Comment by Qingmao-w** [comment]
> Wait, we are on 0.53.5 now, why you are still on 0.50?

Because my project cannot start with v0.50.5 or higher, I don't know why. Currently, I can only use v0.50.4, so I opened this issue🫣☹️

**Comment by Qingmao-w** [comment]
It seems that the terminal crashed after adding this sentence,
Can only downgrade the Unocss version to v0.50.4, or delete this code, @antfu 
![error1](https://github.com/unocss/unocss/assets/120553894/921b9d3e-52fe-4341-99f1-bca0a000cc5b)

**Comment by zyyv** [comment]
@Qingmao-w If this line was the problem produce reason, or you can add magic comment skip this code.
e.g.
```html
<!-- @unocss-skip-start -->
<Form.Item>
 ...
</Form.Item>
<!-- @unocss-skip-end -->
```

And i don't what's reason blocked you use UnoCSS latest version.

**Comment by zoeyzhao19** [comment]
I do some test and found it was the `@unocss/transformer-attributify-jsx` that cause the page stuck, and it was introduce by [PR](https://github.com/unocss/unocss/pull/2334/files) which had updated an elementRE, maybe caused a long backtrack in this case

**Comment by Wxw-Gu** [comment]
> @Qingmao-w If this line was the problem produce reason, or you can add magic comment skip this code.
> 
> > This example used uno's version may higher than yours.
> 
> And i don't what's reason blocked you use UnoCSS latest version.
> 
> e.g.
> 
> ```
> 
> <Form.Item>
>  ...
> </Form.Item>
> 
> ```
> 
> https://unocss.dev/guide/extracting#extracting-from-build-tools-pipeline

It works. Please check if it is caused by `@unocss/transformer-attributify-jsx`.

**Comment by Qingmao-w** [comment]
Has been resolved here #3015
